apps/leo_storage/src/leo_storage_handler_object.erl

```diff
@@ -1180,9 +1180,11 @@ get_active_redundancies(_, []) ->
 get_active_redundancies(Quorum, Redundancies) ->
     AvailableNodes = [RedundantNode ||
                          #redundant_node{available = true} = RedundantNode <- Redundancies],
+    UnavailableNodes = [RedundantNode ||
+                         #redundant_node{available = false} = RedundantNode <- Redundancies],
```

Comment:
Why not just having the flag that indicates if there is unavailable node instead of list UnavailableNodes?